pom.xml

```diff
@@ -706,6 +706,12 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.avro</groupId>
```

Comment:
did we have a previous version of avro soemwhere?

Reply:
No, this is particularly for avro map reduce job. And there was no MR job before. Previously we just print out segment name as the output of hadoop job.